Source/Blazorise/Components/DatePicker/DatePicker.razor.cs

```diff
@@ -571,6 +586,11 @@ protected override IReadOnlyList<TValue> InternalValue
     /// </summary>
     [Parameter] public string DisplayFormat { get; set; }
 
+    /// <summary>
+    /// Defines the input format mask of the date input.
+    /// </summary>
+    [Parameter] public string InputFormat { get; set; }
```

Comment:
Any place we can tell users to easily refer to for formats?

Reply:
Added to the docs.